flink-table-store-core/src/test/java/org/apache/flink/table/store/file/operation/FileStoreCommitTestBase.java

```diff
@@ -71,7 +67,7 @@ public void beforeEach() throws IOException {
         root.getFileSystem().mkdirs(new Path(root + "/snapshot"));
     }
 
-    protected abstract String getSchema();
+    protected abstract String getScheme();
```

Comment:
`getSchema`?

Reply:
Scheme. This is the file system scheme, not table schema.